docs/user/exercises/file-upload.rst

```diff
@@ -1,2 +1,75 @@
-File upload exercise
+File Upload Exercise
 ====================
+
+.. contents:: Content of this document
+    :local:
+    :depth: 2
+
+
+Overview
+--------
+
+Conducting a file exercise consists of 3 steps distributed among
+instructor, Artemis and students:
```

Comment:
The wording here is odd imo. Why not just `Conducting a file exercise consists of 3 steps:`?

Reply:
Yep that sounds more concise, I will use your wording :)